pkg/router/template/router.go

```diff
@@ -1007,6 +1027,35 @@ func (r *templateRouter) createServiceAliasConfig(route *routev1.Route, backendK
 	return &config
 }
 
+func getHeadersList(httpHeaderList []routev1.RouteHTTPHeader) []HTTPHeader {
+	var httpHeadersList []HTTPHeader
+	for _, value := range httpHeaderList {
+		if value.Action.Type == routev1.Set && value.Action.Set != nil && len(value.Name) != 0 {
+			setHeaderValue := sanitizeHeaderValue(value.Action.Set.Value)
+
+			setValue := HTTPHeader{
+				Name:   value.Name,
+				Value:  setHeaderValue,
+				Action: string(value.Action.Type),
+			}
+			httpHeadersList = append(httpHeadersList, setValue)
+		}
+		if value.Action.Type == routev1.Delete && len(value.Name) != 0 {
+			deleteValue := HTTPHeader{
+				Name:   value.Name,
+				Action: string(value.Action.Type),
+			}
+			httpHeadersList = append(httpHeadersList, deleteValue)
+		}
+	}
+	return httpHeadersList
+}
+
+func sanitizeHeaderValue(headerValue string) string {
+	santizeHeaderValue := strings.ReplaceAll(headerValue, "'", `'\''`)
+	return "'" + santizeHeaderValue + "'"
```

Comment:
```suggestion
	sanitizeHeaderValue := strings.ReplaceAll(headerValue, "'", `'\''`)
	return "'" + sanitizeHeaderValue + "'"
```

Reply:
Hi @Miciah ! I dont get what is changed here.

Reply:
s a n t i z e is a typo